src/schema/v1/me/__tests__/record_artwork_view.test.js

```diff
@@ -19,7 +19,7 @@ describe("Recording artwork views", () => {
   `
 
   const typeDefs = fs.readFileSync(
-    path.resolve(__dirname, "../../../data/gravity.graphql"),
+    path.resolve(__dirname, "../../../../data/gravity.graphql"),
```

Comment:
I left this one with the relative path imports but just updated it to point to the right place.